src/osdc/Objecter.h

```diff
@@ -1133,7 +1133,7 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 
   PerfCounters *logger;
 
-  uint64_t tick_event;
+  atomic_t tick_event;
```

Comment:
We do have a 'suspend' method we can use to kill the thread. In the longer term I'd like to move away from having 'init/shutdown/ as separate things from constructor/destructor, so I haven't used it in this case.

I'll get rid of the atomic_t and the assertion, though.